zkevm-circuits/src/evm_circuit/execution/dummy.rs

```diff
@@ -0,0 +1,67 @@
+use std::marker::PhantomData;
+
+use crate::evm_circuit::{
+    execution::ExecutionGadget,
+    step::ExecutionState,
+    util::{constraint_builder::ConstraintBuilder, CachedRegion, Word},
+    witness::{Block, Call, ExecStep, Transaction},
+};
+use crate::util::Expr;
+use eth_types::Field;
+use eth_types::ToLittleEndian;
+use halo2_proofs::plonk::Error;
+
+#[derive(Clone, Debug)]
+pub(crate) struct DummyGadget<F, const N_POP: usize, const N_PUSH: usize, const S: ExecutionState> {
+    pops: [Word<F>; N_POP],
+    pushes: [Word<F>; N_PUSH],
+    _marker: PhantomData<F>,
+}
+
+impl<F: Field, const N_POP: usize, const N_PUSH: usize, const S: ExecutionState> ExecutionGadget<F>
+    for DummyGadget<F, N_POP, N_PUSH, S>
+{
+    const NAME: &'static str = "DUMMY";
+
+    const EXECUTION_STATE: ExecutionState = S;
+
+    fn configure(cb: &mut ConstraintBuilder<F>) -> Self {
+        log::debug!(
```

Comment:
I think it's much better to use a `warn` here, because using the `DummyGadget` means a gadget is not implemented and we're skipping it, so there should be an alert for it.